backend/routes/health.py

```
@@ -0,0 +1,57 @@
+async def status(state: dict = Depends(get_app_state)):
+    """Simple status check."""
+    return {
+        "status": "online",
+        "timestamp": time.time(),
+        "primary_llm": AppConfig.PRIMARY_LLM,
+        "app_initialized": state["app_initialized"],
+    }
+
+
+@router.get("/health")
+async def health_check(state: dict = Depends(get_app_state)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)